gojira.sh

```diff
@@ -572,7 +579,11 @@ function p_compose {
       tfile=$(mktemp /tmp/gojira-egg.yml.XXXXXXXXX)
       $egg > $tfile
       tmps+=("$tfile")
-      flags+=("-f $tfile")
+
+      # Ensure file has data
+      if [[ -s $tfile ]]; then
+        flags+=("-f $tfile")
+      fi
```

Comment:
I guess this check is to ensure an egg from not rendering anything, but I wonder if we are hiding an issue here instead of enforcing a pattern.

No strong opinions on the check, but as far as possible we should try for the egg to not be added unless it will do something. Otherwise, if it's always added then we will always create a tmp file and delete it, which will contribute to the slowing down of every call.